adoc/deployment.adoc

```diff
@@ -24,31 +24,44 @@ To obtain the credentials from {scca}, follow these steps:
 . Select `Proxies` in the top menu. The credentials are displayed in the top right corner.
 . To see the password, click the 'eye' icon.
 . Create a `password.txt` file containing the obtained password.
+[source]
+  $ head -1 ./password.txt | helm registry login registry.suse.com \
+  --username <SUSE_REGISTRY_USERNAME> --password-stdin
+. Create a namespace for {sregistry}.
+[source]
+$ kubectl create namespace <SUSE_REGISTRY_NAMESPACE>
 . Store the mirroring credentials retrieved from {scca} as {kube} secrets by running the following command:
 [source]
   $ kubectl create secret docker-registry suse-registry \
-  --docker-server=registry.suse.com --docker-username=<SUSE_REGISTRY_USERNAME> \
+  --namespace <SUSE_REGISTRY_NAMESPACE> \
+  --docker-server=registry.suse.com \
+  --docker-username=<SUSE_REGISTRY_USERNAME> \
   --docker-password=$(head -1 ./password.txt)
+. Optionally, to use TLS encrypted communication, create a TLS secret from your private key and certificate files.
+[source]
+  $ kubectl create secret tls suse-registry-tls \
+  --namespace <SUSE_REGISTRY_NAMESPACE> \
+  --cert=<CERTIFICATE>.pem \
+  --key=<PRIVATE_KEY>.pem
 
 [#pr-deployment-installing-and-running]
 === Installing and running {productnameshort} using {helm}
 The following procedure describes how to install {productnameshort} using {helm}.
 Replace `<RELEASE_NAME>` with your custom release name for the {helm} chart deployment.
 
 . Log in to {sregistry} using the obtained {scca} mirroring credentials.
-[source]
-  $ head -1 ./password.txt | helm registry login registry.suse.com \
-  --username <SUSE_REGISTRY_USERNAME> --password-stdin
 . Install the latest version of the {productnameshort} {helm} chart.
-  To change the default installation values, refer to <<pr-helm-chart-override>>.
+  To override the default installation with custom values from the `suse_registry_override.yaml` file`, refer to <<pr-helm-chart-override>>.
 [source]
   $ helm install <RELEASE_NAME> \
-  oci://registry.suse.com/private-registry/private-registry-helm
-. The above `helm  install ...` command starts deploying several related containers.
-  The deployment may take several minutes to complete.
+  oci://registry.suse.com/private-registry/private-registry-helm \
+  --namespace <SUSE_REGISTRY_NAMESPACE> \
+  -f suse_registry_override.yaml
```

Comment:
```suggestion
```
If the values aren't there, I think it could confuse readers. It might make sense not to include them. What do you think?